jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/MapUtil.java

```diff
@@ -42,23 +42,21 @@ public static void mergeIfAbsent(Map<String, String> map, Map<String, String> to
     }
 
     /**
-     * Returns a new map with all the entries of map1 and any from map2 which don't override map1.
+     * Returns a new map with all the entries of first map with rest map entries which don't override map1.
      *
-     * Can handle either maps being null. Always returns a new mutable map
+     * Can handle either maps being null. Always returns a new mutable map.
      *
-     * @param map1 first hash map
-     * @param map2 second hash map
+     * @param maps var arg for maps
      * @param <K> first type
      * @param <V> second type
```

Comment:
Should be changed.

We might also want to comment on the precedence of the maps

Reply:
Are you referring to `@param` entries for types?

Reply:
yes